website/docs/ref/macro.md

```diff
@@ -4,6 +4,10 @@
 
 ## Installation
 
+:::tip
+For those who prefer not to use Babel, Lingui offers the [SWC Plugin](/docs/ref/swc-plugin.md) as an alternative.
+:::
+
```

Comment:
I think this should be at the end of the first paragraph of this article. It would make it more clear that Lingui provides Babel macros and if you don't want that, you can use the SWC plugin.

Reply:
fixed

Reply:
Im not agree. People don't read docs thoroughly, when they in the beginning will see "install babel macros" they will close page and will go for searching another lib which is not depending on babel. Note about SWC should be in the place where babel is proposed to be installed.